pkg/controller/deployment/util/deployment_util_test.go

```diff
@@ -960,19 +960,19 @@ func TestDeploymentComplete(t *testing.T) {
 		expected bool
 	}{
 		{
```

Comment:
> hence both rollout status and ProgressDeadlineSeconds will not be successful in cases where a 1-pod Deployment never becomes successful because its Pod is never run

Add a test case for this